agent/base

```diff
@@ -273,28 +274,28 @@ function vercmp {
     local _exp_ver="${3}"
     local _ins_ver="${4}"
 
-	local _stderr=$(rpmdev-vercmp "${_ins_ver}" "${_exp_ver}" 2>&1 > /dev/null)
-	local _res=${?}
-	if [[ ${_res} -ne 0 && ${_res} -ne 11 && ${_res} -ne 12 ]]; then
-		error_log "rpmdev-vercmp - ${_stderr}"
-		return 1
-	fi
-	if [[ "${_match}" == "equ" ]]; then
-		[[ ${_res} -eq 0 ]]
-	elif [[ "${_match}" == "gte" ]]; then
-		[[ ${_res} -eq 0 || ${_res} -eq 11 ]]
-	elif [[ "${_match}" == "gtr" ]]; then
-		[[ ${_res} -eq 11 ]]
-	else
-		error_log "unrecognized 'match' argument, '${_match}'"
-		return 1
-	fi
-	if [[ ${?} -ne 0 ]]; then
-		error_log "wrong version installed: found ${_package}-${_ins_ver}, expected ${_package}-${_exp_ver} (${_match})"
-		return 1
-	fi
-
-	debug_log "found ${_package}-${_ins_ver}"
+    local _stderr=$(rpmdev-vercmp "${_ins_ver}" "${_exp_ver}" 2>&1 > /dev/null)
+    local _res=${?}
+    if [[ ${_res} -ne 0 && ${_res} -ne 11 && ${_res} -ne 12 ]]; then
+	error_log "rpmdev-vercmp - ${_stderr}"
+	return 1
+    fi
+    if [[ "${_match}" == "equ" ]]; then
+	[[ ${_res} -eq 0 ]]
+    elif [[ "${_match}" == "gte" ]]; then
+	[[ ${_res} -eq 0 || ${_res} -eq 11 ]]
+    elif [[ "${_match}" == "gtr" ]]; then
+	[[ ${_res} -eq 11 ]]
+    else
+	error_log "unrecognized 'match' argument, '${_match}'"
+	return 1
+    fi
+    if [[ ${?} -ne 0 ]]; then
+	error_log "wrong version installed: found ${_package}-${_ins_ver}, expected ${_package}-${_exp_ver} (${_match})"
+	return 1
+    fi
+
+    debug_log "found ${_package}-${_ins_ver}"
     return 0
```

Comment:
You only "fixed" the control expression lines: this whole function was indented using tabs, and the lines in between are now just one tab instead of two; which, in "correct" GitHub rendering makes the visual indentation the same as the control lines.

And why *did* you change this file? So far as I can see, there's only the indentation changes, bracketed variables (which I like but I believe isn't essential in any of these contexts), and the addition of `local` declarations ... again, nice but not functional. If there are any "real" changes I'm afraid I lost them in the drizzle of red and green splotches. 😆 Not objecting (except that you should be consistent about the indentation)... but if I did miss anything "functional", please point it out!

Reply:
The changes here are posted in a separate commit.

The changes are all cosmetic, though the use of local declarations is arguably not cosmetic.

I'll post this as a separate PR.

Reply:
Pulled out in PR #2712.